hubpress/scripts/vendor.js

```diff
@@ -69,12 +68,19 @@
           if (this.readyState == 4) {
             if (this.status >= 200 && this.status < 300 || this.status === 304) {
               cb(null, raw ? this.responseText : this.responseText ? JSON.parse(this.responseText) : true, this);
+            } else if (this.status === 401) {
+            	var tfc = prompt("Enter your temporary two-factort authentication code from your app.");
```

Comment:
I was going to fix this but forgot before creating a pull request. Thanks for the reminder! :+1: